web/src/topic_list.ts

```diff
@@ -335,27 +335,23 @@ export function initialize({
 }: {
     on_topic_click: (stream_id: number, topic?: string) => void;
 }): void {
-    $("#stream_filters").on(
-        "click",
-        ".sidebar-topic-check, .topic-name, .topic-markers-and-controls",
```

Comment:
Have you investigated all the possible side-effects of removing `.topic-markers-and-controls` here? I believe this is going to change current behavior for, say, clicking on an @ symbol for a mention.